Source/WebCore/css/CSSComputedStyleDeclaration.cpp

```diff
@@ -55,7 +56,7 @@ CSSComputedStyleDeclaration::CSSComputedStyleDeclaration(Element& element, bool
         name = name.substring(1);
     if (name.startsWith(':'))
         name = name.substring(1);
-    m_pseudoElementSpecifier = CSSSelector::pseudoId(CSSSelector::parsePseudoElementType(name));
+    m_pseudoElementSpecifier = CSSSelector::pseudoId(CSSSelector::parsePseudoElementType(name, CSSSelectorParserContext { element.document() }));
```

Comment:
What does it mean that we return a plain `PseudoId` for the pseudo-elements that take an argument here? It's right (according to CSOSM) that we can do something like `getComputedStyle("::view-transition-group(xxx)")` but I don't think we'll return any computed styles set by such rules in style sheets.

Reply:
Yeah this code is currently broken for both CSS highlights and view transition pseudos. https://bugs.webkit.org/show_bug.cgi?id=264103 was filed to fix this.